docs/Operators.md

```diff
@@ -6911,7 +6911,7 @@ This version of the operator has been available since version 9 of the default O
 #### Attributes
 
 <dl>
-<dt><tt>axes</tt> : list of ints</dt>
+<dt><tt>axes</tt> : list of ints (default is ['0', '2', '3'])</dt>
```

Comment:
The following documentation line "The default is to reduce over all the dimensions of the input tensor." sounds contradictory. Please delete that line as well.